projects/plugins/protect/src/js/routes/scan/scan-footer.jsx

```diff
@@ -125,18 +126,22 @@ const FooterInfo = () => {
 
 const ScanFooter = () => {
 	const { waf } = window.jetpackProtectInitialState || {};
-	return waf.wafSupported ? (
-		<SeventyFiveLayout
-			main={ <ProductPromotion /> }
-			secondary={ <FooterInfo /> }
-			preserveSecondaryOnMobile={ true }
-		/>
-	) : (
-		<Container horizontalSpacing={ 0 } horizontalGap={ 0 } fluid={ false }>
-			<Col>
-				<FooterInfo />
-			</Col>
-		</Container>
+	return (
+		<AdminSectionHero>
+			{ waf.wafSupported ? (
+				<SeventyFiveLayout
+					main={ <ProductPromotion /> }
+					secondary={ <FooterInfo /> }
+					preserveSecondaryOnMobile={ true }
+				/>
+			) : (
+				<Container horizontalSpacing={ 0 } horizontalGap={ 0 } fluid={ false }>
+					<Col>
+						<FooterInfo />
+					</Col>
+				</Container>
+			) }
+		</AdminSectionHero>
```

Comment:
```suggestion
	return waf.wafSupported ? (
		<SeventyFiveLayout
			main={ <ProductPromotion /> }
			secondary={ <FooterInfo /> }
			preserveSecondaryOnMobile={ true }
		/>
	) : (
		<Container horizontalSpacing={ 0 } horizontalGap={ 0 } fluid={ false }>
			<Col>
				<FooterInfo />
			</Col>
		</Container>
	);
```

Per the designs, the footer should remain white.

Reply:
I'll go ahead make this change, but as I mentioned [here](https://github.com/Automattic/jetpack/pull/39702#discussion_r1801730181) this is pattern we use on the Firewall page. Should we maintain that there, or should both changed to have a grey header, white body, and white footer?